02-atmos/components/terraform/fetch-weather/.terraform.lock.hcl

```diff
@@ -0,0 +1,9 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
lock files are OS/arch specific. I think we should leave them out.

Reply:
I haven't gotten the chance to actually use them yet... I didn't not know they were OS specific. That seems to defeat their purpose and is surprising to me. I'll get them removed.